src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -48,6 +50,11 @@ public PersonCard(Person person, int displayedIndex) {
         super(FXML);
         this.person = person;
         id.setText(displayedIndex + ". ");
+        if (person.getArchiveStatus().archiveStatus) {
```

Comment:
So you are suggesting that we keep the archived and not archived employees in the same contact list? OR will this list show only archived when passed in a archived list of  employees?

Reply:
When "c-archive-list" command is given, it will only show all archived persons. When "c-list" command is given, it will show all active(unarchived) persons in the list. Thanks

Reply:
Understood!